core/src/main/java/org/geysermc/geyser/network/GameProtocol.java

```diff
@@ -73,9 +71,10 @@ public final class GameProtocol {
         SUPPORTED_BEDROCK_CODECS.add(CodecProcessor.processCodec(Bedrock_v685.CODEC.toBuilder()
             .minecraftVersion("1.21.0/1.21.1")
             .build()));
-        SUPPORTED_BEDROCK_CODECS.add(DEFAULT_BEDROCK_CODEC.toBuilder()
-            .minecraftVersion("1.21.2/1.21.3")
-            .build());
+        SUPPORTED_BEDROCK_CODECS.add(CodecProcessor.processCodec(Bedrock_v686.CODEC.toBuilder()
+            .minecraftVersion("1.21.2")
```

Comment:
`1.21.2/1.21.3` for version command and whatnot?

Reply:
Ok.

Reply:
https://github.com/GeyserMC/Geyser/pull/4912/commits/523bcdc095a1fb6bf6f6bccca033418a5ad7d92a